src/lib/viewers/BaseViewer.js

```diff
@@ -344,7 +344,10 @@ class BaseViewer extends EventEmitter {
         // Add a resize handler for the window
         document.defaultView.addEventListener('resize', this.debouncedResizeHandler);
 
-        this.addListener('load', () => {
+        this.addListener('load', (event) => {
+            // this.scale set to 1 if event.scale does not exist
+            ({ scale: this.scale = 1 } = event);
```

Comment:
I originally misread what you meant. 

Problem here is that setScale is in Annotator and the eventListener is in BaseViewer.